packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/components/offers/translations/Messages_fr_FR.json

```
@@ -0,0 +1,16 @@
+{
+  "dedicatedCloud_offer_classic": "Veeam Backup Standard",
```

```suggestion
  "dedicatedCloud_datacenter_backup_offer_classic": "Veeam Backup Standard",
```

all translations must start with "dedicatedCloud_datacenter_backup_"

---

This is done